src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md

```diff
@@ -0,0 +1,105 @@
+# Prometheus Exporter AspNetCore for OpenTelemetry .NET
+
```

Comment:
Most of our projects have a bit of a description here about the package. Suggestion:

```suggestion

[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Exporter.Prometheus.AspNetCore.svg)](https://www.nuget.org/packages/OpenTelemetry.Exporter.Prometheus.AspNetCore)
[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Exporter.Prometheus.AspNetCore.svg)](https://www.nuget.org/packages/OpenTelemetry.Exporter.Prometheus.AspNetCore)

An [OpenTelemetry Prometheus exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/prometheus.md) for configuring an ASP.NET Core application with an endpoint for Prometheus to scrape.

```